mesonbuild/modules/rust.py

```diff
@@ -245,6 +247,24 @@ def bindgen(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> Modu
 
         return ModuleReturnValue([target], [target])
 
+    def bindgen_version(self, state: ModuleState, args: T.List, kwargs: FuncBindgen) -> str:
+        """Returns bindgen's version"""
+        if self._bindgen_version is not None:
+            return self.bindgen_version
+
+        if self._bindgen_bin is None:
+            self._bindgen_bin = state.find_program('bindgen')
+
+        cmd = self._bindgen_bin.get_command() + ['--version']
+        try:
+            p, out, err = Popen_safe(cmd)
+            # output begins with 'bindgen '
+            self._bindgen_version = out[8:]
+        except OSError as e:
+            popen_exceptions[join_args(cmd)] = e
+
+        print(self._bindgen_version)
+        return self._bindgen_version
```

Comment:
I think we can actually use `self.bindgen.get_version()` here to get the version, and I think that caches internally so we wouldn't have to cache here.